content/en/sensitive_data_scanner/guide/best_practices_for_creating_custom_rules.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Best Practices for Creating Custom Rules
+disable_toc: false
+further_reading:
+- link: "/sensitive_data_scanner/"
+  tag: "Documentation"
+  text: "Set up Sensitive Data Scanner"
+- link: "/sensitive_data_scanner/regular_expression_syntax"
+  tag: "Documentation"
+  text: "Regular expression syntax for custom rules"
+---
+
+## Overview
+
+Sensitive Data Scanner uses scanning rules to identify, tag, and optionally redact sensitive data in your logs, APM events, and RUM events. Use [out-of-the-box scanning rules][3] or create your own custom rules using [regular expression][1] (regex) patterns. This guide goes over best practices for creating custom rules with regex.
+
+## Use precise regex patterns
+
+Define regex patterns that are as precise as possible because generic patterns result in more false positives. To refine your regex pattern, add test data in the sample data tester when you are creating a custom rule. See step 2 in [Add a custom scanning rule][2] for more information.
```

Comment:
```suggestion
Define regex patterns that are as precise as possible, because generic patterns result in more false positives. To refine your regex pattern, add test data in the sample data tester when creating a custom rule. For more information, see step 2 in [Add a custom scanning rule][2].
```